lusidtools/cocoon/cocoon.py

```diff
@@ -620,8 +620,11 @@ def _convert_batch_to_models(
          A list of populated LUSID request models
     """
 
+    source_columns = [
```

Comment:
Update the type hint for property columns to be list[dict]

Reply:
This might be relevant in a few places where you have changed the type.

Reply:
Good idea, will do that.

Reply:
I did try this, but then it breaks some tests:

```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages/parameterized/parameterized.py", line 533, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/home/runner/work/lusid-python-tools/lusid-python-tools/tests/integration/cocoon/test_cocoon_portfolios.py", line 238, in test_load_from_data_frame_a_portfolios_success
    sub_holding_keys_scope=sub_holdings_key_scope,
  File "/home/runner/work/lusid-python-tools/lusid-python-tools/lusidtools/cocoon/utilities.py", line 92, in _f
    return function(*args, **kwargs)
  File "/home/runner/work/lusid-python-tools/lusid-python-tools/lusidtools/cocoon/cocoon.py", line 1617, in load_from_data_frame
    property_columns=property_columns,
  File "/home/runner/work/lusid-python-tools/lusid-python-tools/lusidtools/cocoon/utilities.py", line 81, in _f
    not isinstance(argument_value, argument_details.annotation)
  File "/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/typing.py", line 716, in __instancecheck__
    return self.__subclasscheck__(type(obj))
  File "/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/typing.py", line 724, in __subclasscheck__
    raise TypeError("Subscripted generics cannot be used with"
TypeError: Subscripted generics cannot be used with class and instance checks
```

Reply:
Okay fair enough.